LiteCore/RevTrees/VectorRecord.cc

```diff
@@ -276,7 +275,7 @@ namespace litecore {
                 revDict[kRevIDKey].setData(newRev.revID);
                 _changed = true;
             }
-            if ( newRev.properties != revDict[kRevPropertiesKey] ) {
+            if ( newRev.properties != revDict.get(kRevPropertiesKey) ) {
```

Comment:
Wondering why using subscript is not recommended?

Reply:
Because fleece::MutableDict subscript operator has a different return type than expected. While fleece::Dict `[]` has a return of `fleece::Value`, and the `get` methods from both Dict and MutableDict return a `fleece::Value`, the MutableDict `[]` returns a `keyref<MutableDict,slice>`. 

The `keyref` type does inherit from `Value`, but has some additional state. Here, it is just used as a `Value`, so `.get` is more suitable.